app/devine/chargepoint/chargepoint_api.py

```diff
@@ -0,0 +1,201 @@
+    for i in status:
+        num_ports = len(i['Port'])
+        ports_inuse = []
+        ports_notuse = []
+        for p in i['Port']:
+            # print('-'*100)
+            # print(p['Status'])
+            if p['Status'] == 'INUSE':
+                ports_inuse.append([p['portNumber'],p['Status'], p['Power'], p['TimeStamp']])
+            else:
+                ports_notuse.append([p['portNumber'],p['Status'], p['Power'], p['TimeStamp']])
+        if len(ports_inuse) > 0:
+            stations_inuse[i['stationID']] = ports_inuse
+        if len(ports_notuse) > 0:
+            stations_notuse[i['stationID']] = ports_notuse
+    ii=0
+    retval = []
+    station_load_map = {} #save the station_load to avoid getLoad again when ports not use
+    for i in stations_inuse:
+        station_load,ret = cp.getStationLoad(queryID=i)
+        station_data = ret['stationData'][0]
+        station_load_map[i] = station_load
+        load = []
+        for p in range(len(stations_inuse[i])): #how many ports
+            port_num = int(stations_inuse[i][p][0]) #the port#
+            port_status = stations_inuse[i][p][1]
+            port_power = stations_inuse[i][p][2]
+            port_timestamp = stations_inuse[i][p][3]
+            load.append(parsePort(station_data['Port'][port_num-1], port_num, port_status, port_power, port_timestamp))
+            # parsePort(station_data['Port'][port_num-1], port_status)
+        retval.append({'station_id':i, 'station_load':station_load, 'Port':load})
+        ii+=1
+        print(name[1]+str(ii))
+
+    for j in stations_notuse:
+        station_load_notuse = 0.000
+        if j in station_load_map:
+            station_load_notuse = station_load_map[j]
+        load_notuse = []
+        for q in range(len(stations_notuse[j])):
+            port_num_notuse = int(stations_notuse[j][q][0]) #the port#
+            port_status_notuse = stations_notuse[j][q][1]
+            port_power_notuse = stations_notuse[j][q][2]
+            port_timestamp_notuse = stations_notuse[j][q][3]
+            data_notuse={
+                'shedState': 0,
+                'portLoad':0.000,
+                'allowedLoad':0.000,
+                'userID': '',
+            }
+            load_notuse.append(parsePort(data_notuse, port_num_notuse, port_status_notuse, port_power_notuse, port_timestamp_notuse))
+        retval.append({'station_id':j, 'station_load':station_load_notuse, 'Port':load_notuse})
+        
+    # print(retval)
+    return retval
+
+def read_from_sites():
+    start = time.time()
+    sites=[['slac-gismo', 'slac_GISMO_sgID'],
+           ['slac', 'slac_B53_sgID'],
+           ['google', 'google_CRIT_sgID'],
+           ['google', 'google_B46_sgID'],
+           ['google', 'google_plymouth_sgID']]
+    retval = []
+    for site in sites:
+        retval.append(getData(site))
+        print('finish #')
+    # print(retval)
+    print('Total Time: ', time.time()-start)
+    return retval
+
+
```

Comment:
Need consistent naming conventions throughout. You can check out the PEP 8 style guide, which we generally follow. But at a glance, pascal case the class name and snake case the function and variable names.